app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lhi06.residue;
+
+import android.support.annotation.NonNull;
+
+/**
+ * Created by lhi06 on 20/07/2017.
+ */
+
+public class AlbumReview {
+
+    private Album album;
+    private int rating;
+    private String review = "";
+
+    public AlbumReview(@NonNull final Album thisAlbum, final int rating, @NonNull final String review) {
+        this.album = thisAlbum;
+        this.rating = rating;
+        this.review = review;
+    }
+
+    public Album getThisAlbum() {
+        return album;
+    }
+
+    public void setThisAlbum(final Album thisAlbum) {
```

Comment:
The reason for deleting the setters is that, when you create data classes like this one, you should aim the make them immutable (i.e. no way to change them after creation) as immutable data classes are safer to use and should lead to fewer bugs in the code. If you add setters to begin with before you start to use the class then you are making a design decision to make the class mutable and you shouldn't make that decision until you are sure a mutable class is required. It's easy to add the setters in later if you really need them as it's just boilerplate code